scripts/unit-test-setup.js

```diff
@@ -0,0 +1,14 @@
+module.exports = () => {
+  process.env.NX_DAEMON = 'false';
```

Comment:
Maybe add a comment. Turning this to true, doesn't actually mean the source code of the daemon is getting tested. It would mean that the installed version of the daemon is getting tested. Which we do not want.